### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by ibraimgm**

### CODE REVIEW

File: internal/config/config.go [hunk 1]
```diff
@@ -1,16 +1,21 @@
 package config
 
+// Config holdsthe entire application configuration.
```
Comment (cpurta) [inline internal/config/config.go]: Minor typo 'holds the' instead of 'holdsthe'

**Review by cpurta — COMMENTED** [review]
This looks great! There is just the minor typo change that needs to be done and I'll get this merged in once that is done.

File: internal/config/config.go (hunk 1 above)
Comment (ibraimgm) [reply internal/config/config.go]: Done! Sorry for the delay.

**Comment by cpurta** [comment]
This looks great! Thanks for working on this! Merging

**Pull request merged**